app/scripts/metamask-controller.test.js

```diff
@@ -3944,106 +3941,6 @@ describe('MetaMaskController', () => {
-          },
-        },
-      };
-      let localMetamaskController;
-      let spyBalancesTrackerUpdateBalance;
-
-      beforeEach(() => {
-        jest.useFakeTimers();
-        jest.spyOn(MultichainBalancesController.prototype, 'updateBalances');
-        jest
-          .spyOn(MultichainBalancesController.prototype, 'updateBalance')
-          .mockResolvedValue();
-        spyBalancesTrackerUpdateBalance = jest
-          .spyOn(MultichainBalancesTracker.prototype, 'updateBalance')
-          .mockResolvedValue();
-        localMetamaskController = new MetaMaskController({
-          showUserConfirmation: noop,
-          encryptor: mockEncryptor,
-          initState: {
-            ...cloneDeep(firstTimeState),
-            AccountsController: {
-              internalAccounts: {
-                accounts: {
-                  [mockNonEvmAccount.id]: mockNonEvmAccount,
-                  [mockEvmAccount.id]: mockEvmAccount,
-                },
-                selectedAccount: mockNonEvmAccount.id,
-              },
-            },
-          },
-          initLangCode: 'en_US',
-          platform: {
-            showTransactionNotification: () => undefined,
-            getVersion: () => 'foo',
-          },
-          browser: browserPolyfillMock,
-          infuraProjectId: 'foo',
-          isFirstMetaMaskControllerSetup: true,
-        });
-      });
-
-      afterEach(() => {
-        jest.clearAllMocks();
-        jest.useRealTimers();
-      });
-
-      it('calls updateBalances during startup', async () => {
-        expect(
-          localMetamaskController.multichainBalancesController.updateBalances,
-        ).toHaveBeenCalled();
-      });
-
-      it('calls updateBalances after the interval has passed', async () => {
-        // 1st call is during startup:
-        // updatesBalances is going to call updateBalance for the only non-EVM
-        // account that we have
-        expect(
-          localMetamaskController.multichainBalancesController.updateBalances,
-        ).toHaveBeenCalledTimes(1);
-        expect(spyBalancesTrackerUpdateBalance).toHaveBeenCalledTimes(1);
-        expect(spyBalancesTrackerUpdateBalance).toHaveBeenCalledWith(
-          mockNonEvmAccount.id,
-        );
-
-        // Wait for "block time", so balances will have to be refreshed
-        jest.advanceTimersByTime(
-          MULTICHAIN_BALANCES_UPDATE_TIME[BtcAccountType.P2wpkh],
-        );
-
-        // Check that we tried to fetch the balances more than once
-        // NOTE: For now, this method might be called a lot more than just twice, but this
-        // method has some internal logic to prevent fetching the balance too often if we
-        // consider the balance to be "up-to-date"
-        expect(
-          spyBalancesTrackerUpdateBalance.mock.calls.length,
-        ).toBeGreaterThan(1);
-        expect(spyBalancesTrackerUpdateBalance).toHaveBeenLastCalledWith(
-          mockNonEvmAccount.id,
-        );
-      });
-    });
```

Comment:
Removing this because MultichainBalancesController no longer polls to update balances